Makefile

```diff
@@ -244,10 +247,19 @@ run:
 
 .PHONY: docker-component # Not intended to be used directly
 docker-component: check-component
+	GOOS=$(GOOS) GOARCH=$(GOARCH) $(MAKE) $(COMPONENT)
+	cp ./bin/$(COMPONENT)_$(GOOS)_$(GOARCH) ./cmd/$(COMPONENT)/$(COMPONENT)
+	cp -r ./config ./cmd/$(COMPONENT)
+	docker build -t holoinsight/$(COMPONENT):$(TAG) ./cmd/$(COMPONENT)/
+	rm ./cmd/$(COMPONENT)/$(COMPONENT)
+	rm -rf ./cmd/$(COMPONENT)/config
+
+.PHONY: docker-component-multiarch # Not intended to be used directly
+docker-component-multiarch: check-component
 	GOOS=linux GOARCH=amd64 $(MAKE) $(COMPONENT)
```

Comment:
You build collector binary here using current host ARCH.
But build two platform docker images.
These two platform docker images will have same binary with current ARCH.